qa/src/test/resources/java-web-app-and-db-with-function.yaml

```diff
@@ -21,12 +21,13 @@ location: localhost
 services:
 - type: org.apache.brooklyn.entity.webapp.ControlledDynamicWebAppCluster
   name: My Web
-  wars.root: http://repo1.maven.org/maven2/org/apache/brooklyn/example/brooklyn-example-hello-world-sql-webapp/0.9.0/brooklyn-example-hello-world-sql-webapp-0.9.0.war
-  http.port: 9280+
-  proxy.http.port: 9210+
-  java.sysprops: 
-    brooklyn.example.db.url: $brooklyn:formatString("jdbc:%s%s?user=%s&password=%s",
-       component("db").attributeWhenReady("datastore.url"), "visitors", "brooklyn", "br00k11n")
+  brooklyn.config:
```

Comment:
I'm surprised this is needed, but it's most probably a good thing to include the `brooklyn.config`, so happy to merge.